src/core/src/service_interfaces/TelemetryWriter.py

```diff
@@ -13,72 +13,61 @@
 # limitations under the License.
 #
 # Requires Python 2.7+
-
+import datetime
 import json
+import os
+import re
+import shutil
+import tempfile
+import time
+
 from core.src.bootstrap.Constants import Constants
 
 
 class TelemetryWriter(object):
     """Class for writing telemetry data to data transports"""
 
-    def __init__(self, env_layer, execution_config):
-        self.data_transports = []
+    def __init__(self, composite_logger, env_layer, events_folder_path):
```

Comment:
Fix ordering to match other class layouts -- env_layer, composite_logger, events_folder_path